includes/class-wc-mondido-subscriptions.php

```diff
@@ -88,23 +86,14 @@ public function subscription_options_product_tab_content() {
 	 */
 	public function save_subscription_field() {
 		global $post_id;
-		$order = wc_get_order( $post_id );
 
 		if ( empty( $post_id ) ) {
 			return;
 		}
 
 		if ( isset( $_POST['_mondido_plan_id'] ) ) {
-			if ( OrderUtil::custom_orders_table_usage_is_enabled() ) {
-				// HPOS usage is enabled.
-				$order->update_meta_data( '_mondido_plan_id', $_POST['_mondido_plan_id'] );
-				$order->update_meta_data( '_mondido_plan_include', $_POST['_mondido_plan_include'] );
-				$order->save();
-			} else {
-				// Traditional CPT-based orders are in use.
-				update_post_meta( $post_id, '_mondido_plan_id', $_POST['_mondido_plan_id'] );
-				update_post_meta( $post_id, '_mondido_plan_include', $_POST['_mondido_plan_include'] );
-			}
+			update_post_meta( $post_id, '_mondido_plan_id', $_POST['_mondido_plan_id'] );
```

Comment:
Because according to their docs it's only for calls that either access or modify the _order_ data.

"If you see one of the matches are directly accessing or modifying order data, you will need to change it to use WooCommerce's CRUD API instead"

I didn't find an $order object so assumed it wasn't needed. However I will double-check since I think this will be hard to test because it's subscriptions.

Reply:
I did a simple check to make sure that the $post in this case is not an actual order object and it returned false so we should be good to go.